script/publishing-api-sync-checks/case_study_sync_check.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+$LOAD_PATH << File.expand_path("../", File.dirname(__FILE__))
+
+require 'pathname'
```

Comment:
You could remove a bunch of this bootstrapping if you were shelling out with `rails runner` instead. Is there any reason (other than historical) we aren't?